kernel/src/main/java/org/kframework/kast/KastFrontEnd.java

```diff
@@ -125,6 +127,9 @@ public int run() {
             KRead kread = new KRead(kem, files.get(), options.input);
             if (options.genParser || options.genGlrParser) {
                 kread.createBisonParser(parsingMod, sort, options.bisonOutputFile(), options.genGlrParser, options.bisonFile, options.bisonStackMaxDepth);
+                try {
+                  Files.copy(options.bisonOutputFile().toPath(), files.get().resolveKompiled("parser_" + sort.name() + "_" + options.module).toPath());
+                } catch (IOException e) {}
```

Comment:
I just want to double-check. If I re-call `kast ....` to generate a parser, it will overwrite any existing parser that is there, correct?

Reply:
Well, yeah. That's not very likely to happen in practice though unless you're doing it deliberately.